### ISSUE TITLE
[Bug] - MdAutocomplete width of dropdown sometimes too small

### ISSUE THREAD

**Issue by emman27** [vuematerial/vue-material#1239]
### Steps to reproduce

Running autocomplete with a promise based system

### Which browser?
Chrome 62, Ubuntu 16.04

### What is expected?
Full width autocomplete

### What is actually happening?
![screenshot from 2017-12-01 13-37-35](https://user-images.githubusercontent.com/6295583/33469283-efb6faa2-d69c-11e7-8d0c-fbb491166c54.png)

### Reproduction Link
Couldn't replicate the code in a fiddle, but an abridged version:

```
<template>
<div class="md-layout md-gutter">
  <div class="md-layout-item md-small-size-100" v-else>
    <md-autocomplete v-model="entityType" :md-options="entityTypes">
      <label>Entity Type</label>
    </md-autocomplete>
  </div>
</div>
</template>

<script>
import getEntityTypes from 'somewhere';

export default {
  data() {
    return {
      entityType: '',
      entityTypes: []
    };
  },
  async mounted() {
    this.entityTypes = await getEntityTypes();
  }
};
</script>
```

Am I doing something wrong?

**Comment by Samuell1** [comment]
It can be because its waiting on api and then its not recalculating width? or because you forgot to remove `v-else`

**Comment by emman27** [comment]
@Samuell1 Most definitely not, the API resolves cleanly. The v-else is a typo whoops. In my actual codebase I have a v-if preceding it correctly.

**Comment by Samuell1** [comment]
@emman27 i think it can be because you calling api in mounted and thats after rendering, try use `created` hook

**Comment by emman27** [comment]
@Samuell1 Hey thanks, great catch. Unfortunately doesn't resolve the issue, though it does flag out some problems in my codebase.

**Comment by Samuell1** [comment]
@emman27 Hey, do you still have this issue? I cant reproduce it.

**Comment by emman27** [comment]
@Samuell1 Yup it's not happening to me anymore. Fix by coincidence with some other issue perhaps? I think we can close this issue :)
